### PR TITLE
Real types in tasks and metrics

### PR DESCRIPTION
**Pull request by elronbandel**

### CODE REVIEW

File: prepare/cards/coedit.py [hunk 1]
```diff
@@ -120,7 +120,8 @@
                 "input_type": "sentence",
             }
         ),
-        RenameFields(field_to_field={"src": "input", "tgt": "output_choice"}),
+        RenameFields(field="src", to_field="input"),
+        IndexOf(search_in="choices", index_of="tgt", to_field="output_choice"),
```
Comment (yoavkatz) [inline prepare/cards/coedit.py:124]: Is this a bug fix? Should be noted in change log.

File: src/unitxt/operator.py [hunk 2]
```diff
@@ -135,8 +135,12 @@ class MultiStreamOperator(StreamingOperator):
 
     caching: bool = NonPositionalField(default=None)
 
-    def __call__(self, multi_stream: Optional[MultiStream] = None) -> MultiStream:
+    def __call__(
+        self, multi_stream: Optional[MultiStream] = None, **kwargs
```
Comment (yoavkatz) [inline src/unitxt/operator.py:139]: How is this change related? What does it do? Not clear.

Comment (elronbandel) [reply src/unitxt/operator.py:139]: Completely unrelated will delete that

Comment (elronbandel) [reply src/unitxt/operator.py]: done

**Review by yoavkatz — APPROVED** [review]

**Review by pawelknes — APPROVED** [review]
L
